modules/light-clients/07-tendermint/types/upgrade_test.go

```diff
@@ -452,9 +452,6 @@ func (suite *TendermintTestSuite) TestVerifyUpgrade() {
 		cs := suite.chainA.GetClientState(path.EndpointA.ClientID)
 		clientStore := suite.chainA.App.GetIBCKeeper().ClientKeeper.ClientStore(suite.chainA.GetContext(), path.EndpointA.ClientID)
 
-		// Call ZeroCustomFields on upgraded clients to clear any client-chosen parameters in test-case upgradedClient
```

Comment:
this caused a test an existing test case to pass `unsuccessful upgrade: committed client does not have zeroed custom fields`, this test now passes due to the zeroing in `VerifyUpgradeAndUpdateState`